playground/.env.example

```diff
@@ -0,0 +1,6 @@
+ETH_RPC_URL=http://erigon.dappnode:8545
```

Comment:
Would be nice to mention `http://host.docker.internal:8545` here if you are running a node locally.

Reply:
I think it's better to discourage this as per the above.